src/pkcs11/pkcs11-global.c

```diff
@@ -485,6 +486,19 @@ CK_RV C_GetSlotInfo(CK_SLOT_ID slotID, CK_SLOT_INFO_PTR pInfo)
 
 	sc_log(context, "C_GetSlotInfo(0x%lx)", slotID);
 
+	if (sc_pkcs11_conf.plug_and_play)
+		uninit_slotcount = 1;
```

Comment:
I personally prefer braces with all conditionals (as per https://www.securecoding.cert.org/confluence/display/c/EXP19-C.+Use+braces+for+the+body+of+an+if%2C+for%2C+or+while+statement) but as there is no consistent style in OpenSC, it doesn't really matter...